### DISCUSSION TITLE
刪除應用程式殘存檔案工具分享

### DISCUSSION THREAD

**Discussion by xkllkx** [1101-GS4719-C/Python-Community-GS4719-C#429] [General]
有時因為硬碟壞掉、不小心移動到應用程式的路徑或是誤刪到應用程式的檔案，都會導致應用程式無法順利執行，但又因為尚有殘存檔案無法卸載，重新安裝，真的會搞到心很累，不彷可以嘗試使用這個工具，經過測試，它真的能把之前殘存的檔案刪除乾淨。
[Revo Uninstaller](https://www.kocpc.com.tw/archives/319794)

**Comment by pekoranoinu** [comment]
感謝樓主分享，殘留得檔案真的是非常惱人。

**Comment by ghost** [comment]
感謝教學 已下載<3，來看一下有什麼沒刪乾淨

**Comment by chupochun** [comment]
剛剛已經去下載了但是想知道有沒有不用付費的免費版呢
